### PR TITLE
Add option to channel resampler to resample to detector sampling rate + refactoring

### PR DESCRIPTION
**Pull request by fschlueter**

### CODE REVIEW

File: NuRadioReco/modules/channelResampler.py [hunk 1]
```diff
@@ -39,12 +23,15 @@ def run(self, evt, station, det, sampling_rate):
 
         evt, station, det
             Event, Station, Detector
-        sampling_rate: float
+        sampling_rate: float (Default: None)
             In units 1/time provides the desired sampling rate of the data.
```
Comment (sjoerd-bouma) [inline NuRadioReco/modules/channelResampler.py:27]: This is a neat feature! Can you add what happens if sampling_rate=None to the docstrings?

Comment (fschlueter) [reply NuRadioReco/modules/channelResampler.py]: fine?

File: NuRadioReco/modules/channelResampler.py [hunk 2]
```diff
@@ -8,43 +8,30 @@ class channelResampler:
...
         Parameters
         ----------
 
         evt, station, det
             Event, Station, Detector
-        sampling_rate: float
-            In units 1/time provides the desired sampling rate of the data.
-
+        sampling_rate: float (Default: None)
+            In units `1 / time` provides the desired sampling rate of the data.
```
Comment (sjoerd-bouma) [inline NuRadioReco/modules/channelResampler.py:27]: I'm not sure what a single \` does, but for `single-spaced` text you need two of them (if that's what you were going for - not sure how this renders currently, usually I think single backquote imply crossreferences?)

Comment (fschlueter) [reply NuRadioReco/modules/channelResampler.py]: better?

**Comment by fschlueter** [comment]
jipppi!

**Pull request merged**